src/app/component/ProductPrice/ProductPrice.style.scss

```diff
@@ -10,8 +10,8 @@
  */
 
 :root {
-    --price-color: #000;
-    --price-discount-color: #000;
+    --price-color: #b10d0d;
+    --price-discount-color: #bcb9b9;
```

Comment:
Consider using the same color notation that is used throughout the application. I believe we use rgb[a]